tf-101/4-variables.md

```diff
@@ -26,19 +26,19 @@ redirect: https://developer.oracle.com/tutorials/tf-101/4-variables/
 {% slides %}
 {% imgx aligncenter assets/terraform-101.png 400 400 "Terraform 101" "Terraform 101 Tutorial Series" %}
 
-This lesson will take a deeper look at Terraform variabales.  You've already had a little bit of exposure to them in the [Experiencing Terraform](2-experiencing-terraform) tutorial, as well as a brief summary in the [Understanding Terraform Concepts](3-understanding-terraform) tutorial.  Let's dive in and take a deeper look at what these are and how they're used.
+This lesson will take a deeper look at Terraform variabales. You've already had a little bit of exposure to them in the [Experiencing Terraform](2-experiencing-terraform) tutorial, as well as a brief summary in the [Understanding Terraform Concepts](3-understanding-terraform) tutorial. Let's dive in and take a deeper look at what these are and how they're used.
 
 ## Why use variables?
 
-Variables provide a way to easily decouple a value and its reference from your Terraform code.  A topology may be defined, but the specifics are given programmatically (or manually).  Writing parameterized Terraform code (using variables for many of the customizable values), means that as an environment changes new values for the variables may be provided without requiring any modification to the underlying Terraform code.
+Variables provide a way to easily decouple a value and its reference from your Terraform code. A topology may be defined, but the specifics are given programmatically (or manually). Writing parameterized Terraform code (using variables for many of the customizable values), means that as an environment changes, new values for the variables may be provided without requiring any modification to the underlying Terraform code.
 
 Here are some common use-cases for parameterizing code:
 
-*	Multiple deployments of the same topology.  This could be for deploying separate development, staging and production environments that share identical topologies (but have unique names, compartments, CIDR blocks, etc.).
-*	Passing secret/sensitive data in via variables.  Rather than hard-coding credentials, keys and other sensitive data in the Terraform code, this can be passed via variables.
-*	Writing extensible modules in Terraform, where variables are used as inputs to the module, determining its behavior and/or setting resource attributes.
+*	Multiple deployments of the same topology—this could be for deploying separate development, staging and production environments that share identical topologies (but have unique names, compartments, CIDR blocks, etc.).
```

Comment:
for what should be em dashes, please use a double dash surrounded by spaces, e.g. `this -- that`